File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/facebook/FacebookPackage.java
package abi43_0_0.expo.modules.facebook;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi43_0_0.expo.modules.core.BasePackage;
import abi43_0_0.expo.modules.core.ExportedModule;

public class FacebookPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new FacebookModule(context));
  }
}

File: android/versioned-abis/expoview-abi45_0_0/src/main/java/abi45_0_0/expo/modules/facebook/FacebookPackage.kt
package abi45_0_0.expo.modules.facebook

import android.content.Context
import abi45_0_0.expo.modules.core.BasePackage
import abi45_0_0.expo.modules.core.ExportedModule

class FacebookPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(FacebookModule(context))
}
